src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenUsingDeclarationTests.cs

```diff
@@ -1403,5 +1403,263 @@ static async Task Main()
 
             CompileAndVerify(compilation, expectedOutput: "Dispose async 0");
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/72496")]
+        public void ModifiersInUsingLocalDeclarations_Const()
```

Comment:
Let's add versions of these that aren't in top-level statements. `using` is parsed differently there vs in method bodies.